### ISSUE TITLE
[Bug]: Chat not showing from server while in MineTogether tab

### ISSUE THREAD

**Issue by gigabit101** [CreeperHost/CreeperHostGui#631]
### Describe the bug

Chat not showing from server while in MineTogether tab

### Steps to reproduce

1 Connect to a dedicated Server.
2 Open the MineTogether chat tab
3 Have someone else on the server send a message, If not available /say command
4 Don't get messages in the game when switching back to normal chat

### Expected behaviour

To see chat

### Version

1.18.2-6.0.7

### Minecraft Version

1.18.2

### ModLoader Version

Forge

**Comment by covers1624** [comment]
I am unable to reproduce this in dev.

Can you re-test once 6.0.9 is out?

**Comment by Raidobw2** [comment]
I have feedback concerning this issue, coming from https://github.com/FTBTeam/FTB-Modpack-Issues/issues/1603.
Crash log: https://pastebin.com/UtufkfB3
Log: https://pastebin.com/DAUwbn2G

How the players reproduced it:

> Another bug related to MineTogether comes when going to the pause menu (Esc key) > Green chat bubble icon at the top right > Try to toggle "Chat" from Enabled to Disabled, the game shuts itself down (shows a "Saving world" screen and then closes the instance)

Useful information also found in the full comment at https://github.com/FTBTeam/FTB-Modpack-Issues/issues/1603#issuecomment-1304696413.

Both are using `minetogether-forge-1.18.2-6.0.8.jar`

Thanks for checking this out!

**Comment by OneComputerGuy** [comment]
A bit more info about this issue, the error is shown/experienced if the socket for MineTogether fails to connect and never starts showing messages.

I tried this morning again, the chat did connect successfully and, only then, I was able to disable the chat and see the server messages.

The messages from the server/say command are shown in MineTogether only if the socket is opened successfully, if it fails, it will not show any messages in the MTogh tab or the Commands tab.

A way to replicate this would be to make the connection fail on Minetogether, so it can get the following error:

`java.lang.IllegalArgumentException: Not connected to server`

Once that is done, try to send server messages or disable the chat, the first option will not show anything on any of the chat tabs and the second option will close the game with the error:

`[05Nov2022 20:56:24.545] [Render thread/ERROR] [net.minecraft.client.Minecraft/FATAL]: Reported exception thrown!
net.minecraft.ReportedException: mouseClicked event handler`

One thing to note that it was different visually from yesterday when it was failing constantly is that, when accessing the pause menu (Esc key) yesterday I was able to see a green bubble chat icon at the top right along with a cogwheel, today, I only see a green cogwheel for the settings
